File: app/src/main/java/edu/com/softserveinc/bawl/dto/UserIssuesHistoryDto.java
package edu.com.softserveinc.bawl.dto;


public class UserIssuesHistoryDto {

    private String issueName ;

    private IssueHistoryDto issueHistoryDto;

    private int currentStatus;

    public String getIssueName() {
        return issueName;
    }

    public void setIssueName(String issueName) {
        this.issueName = issueName;
    }

    public IssueHistoryDto getIssueHistoryDto() {
        return issueHistoryDto;
    }

    public void setIssueHistoryDto(IssueHistoryDto issueHistoryDto) {
        this.issueHistoryDto = issueHistoryDto;
    }

    public int getCurrentStatus() {
        return currentStatus;
    }

    public void setCurrentStatus(int currentStatus) {
        this.currentStatus = currentStatus;
    }






}

File: app/src/main/scala/edu/com/softserveinc/bawl/dto/UserIssuesHistoryDto.scala
package edu.com.softserveinc.bawl.dto

import scala.beans.BeanProperty

class UserIssuesHistoryDTO {
  @BeanProperty  var issueName: String = null
  @BeanProperty  var issueHistoryDto: IssueHistoryDTO = null
  @BeanProperty  var currentStatus: String = ""
}